datalad_debian/bootstrap_builder.py

```diff
@@ -28,15 +28,44 @@
 @build_doc
 class BootstrapBuilder(Interface):
     """Bootstrap a build environment
+
+    This command bootstraps a (containerized) build environment, such as a
+    Singularity container, based on an existing builder configuration, such as
+    a Singularity recipe.
+
+    The execution of this command might require administrative privileges and
+    could prompt for a sudo password, for example in order to build a
+    Singularity image. The resulting bootstrapped build environment will be
+    placed inside of a 'envs/' subdirectory of a 'builder/' dataset-
+
+    The following directory tree illustrates this with an example. The
+    configured builder takes the form of a Singularity recipe here.
+
+        |    bullseye                <- distribution dataset
+        |    ├── builder             <- builder subdataset
+        |    │   ├── envs
+        |    │   │   ├── README.md
+        |    │   │   └── singularity-amd64.sif   <- bootstrapped build environment
+        |    │   └── recipes
+        |    │       ├── README.md
+        |    │       └── singularity-any     <- configured builder recipe
```

Comment:
I'm not a fan of the wording "configured builder recipe".

Above you wrote:

> existing builder configuration, such as a Singularity recipe.

Which I really like.

So this would read cleaner to me if it was "builder recipe" or "builder configuration".